src/pretix/control/templates/pretixcontrol/waitinglist/index.html

```diff
@@ -96,6 +96,7 @@
                     <th>{% trans "On the list since" %}</th>
                     <th>{% trans "Status" %}</th>
                     <th>{% trans "Voucher" %}</th>
+                    <th>{% trans "Delete" %}</th>
```

Comment:
We normally do not label the last column, if it only contains buttons that speak for themselves, so just use ``<th></th>``